client/elife-theme/src/index.js

```diff
@@ -40,14 +42,15 @@ export default {
   fontLineHeight6: '16px',
 
   /* Spacing */
-  gridUnit: '6px',
-  verticalXxl: 'gridUnit'*20,
-  verticalXl: 'gridUnit'*12,
-  verticalL: 'gridUnit'*8,
-  verticalM: 'gridUnit'*6,
-  verticalS: 'gridUnit'*4,
-  verticalXs: 'gridUnit'*2,
-  verticalXxs: 'gridUnit',
+  gridUnit: `${gridUnit}px`,
+  space: [
+    0,
+    gridUnit,
+    gridUnit * 2,
+    gridUnit * 4,
+    gridUnit * 12,
+    gridUnit * 16,
```

Comment:
These multiples are different to those in the design style guide. Is that intentional?

Reply:
No, not at all :flushed:
Thanks :smiley: